test/spec/FileTreeView-test.js

```diff
@@ -29,12 +29,31 @@ define(function (require, exports, module) {
 
     var FileTreeView      = require("project/FileTreeView"),
         FileTreeViewModel = require("project/FileTreeViewModel"),
-        React             = require("thirdparty/react"),
-        ReactDOM          = require("thirdparty/react-dom"),
+        React             = require("thirdparty/preact"),
+        ReactDOM          = require("thirdparty/preact"),
         Immutable         = require("thirdparty/immutable"),
-        RTU               = React.addons.TestUtils,
+        RTU               = require("thirdparty/preact/preact-test-utils"),
         _                 = require("thirdparty/lodash");
 
+    // Preact Test Utils doesn't have findRenderedDOMComponentWithTag method
+    // So create it
+    RTU.findRenderedDOMComponentWithTag = function(root, tagName) {
```

Comment:
Instead of extending the `RTU` object can we just add `findRenderedDOMComponentWithTag` as a helper method?

``` js
function findRenderedDOMComponentWithTag(root, tagName) { ... }

...

findRenderedDOMComponentWithTag(...,...)
```

Just in case `findRenderedDOMComponentWithTag` gets added to RTU in future